assignments/arrays.js

```diff
@@ -63,34 +63,91 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+function findCar (number, arr){
+    let string = "";
+    for(let i = 0; i < arr.length; i++){
+        if(arr[i].id == number) {
+            string = `Car ${arr[i].id} is a ${arr[i].car_year} ${arr[i].car_make} ${arr[i].car_model}`;
+        }
+    }
+   return string;
+}
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(findCar(33, inventory));
 
 
 
-// ==== Challenge 2 ====
+
+//===================================Challenge 2 ===================================================================================================================
+//==================================================================================================================================================================
+//==================================================================================================================================================================
+
+
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// function findLast(arr){
+// let lastCar = "";
+// for (let i = arr.length-1; i > 0; i--){
+//     if(i == arr.length-1) {
+//        lastCar = arr[i].car_make + " " + arr[i].car_model;
+//         }
+//     }
+//     return lastCar;
+// }
+// console.log(findLast(inventory));
+
+lastCar = inventory[49];
+console.log(lastCar.car_make + ' ' + lastCar.car_model);
 
```

Comment:
You can do something like this instead. If another item gets added into the array you won't be grabbing the last item in the array your way.
```suggestion
let last_element = my_array[my_array.length - 1];
```